game/marvin.cpp

```diff
@@ -63,7 +63,7 @@ Marvin::Marvin() {
     {"set var %v %s",              C_SetVar},
 
     // animation
-    {"play ani %s",                C_Invalid},
+    {"play ani %s",                C_PlayAni},
```

Comment:
Addition of `play ani` command seem to be orthogonal to strafe itself. Generally, I'm fine merging both things in one PR - just double checking